content/blog/2018-03-27-update-on-async-rendering.md

```diff
@@ -94,6 +94,10 @@ There is a common misconception that fetching in `componentWillMount` lets you a
 >
 > In the longer term, the canonical way to fetch data in React components will likely be based on the “suspense” API [introduced at JSConf Iceland](/blog/2018/03/01/sneak-peek-beyond-react-16.html). Both simple data fetching solutions and libraries like Apollo and Relay will be able to use it under the hood. It is significantly less verbose than either of the above solutions, but will not be finalized in time for the 16.3 release.
 
+> Note
+>
```

Comment:
I think two notes look a bit odd, maybe we could combine? Although I don’t feel strongly and maybe separate is better. Just calling some extra attention to this.